examples/plot_firing_pattern.py

```diff
@@ -30,26 +29,33 @@
 import matplotlib.pyplot as plt
 
 net = Network(params)
-with NetworkBuilder(net) as network_builder:
-    network_builder.cells[0].plot_voltage()
-
-    # The cells are stored in the network object as a list
-    cells = network_builder.cells
-    print(cells[:5])
-
-    # We have different kinds of cells with different cell IDs (gids)
-    gids = [0, 35, 135, 170]
-    for gid in gids:
-        print(cells[gid].name)
-
-    # We can plot the firing pattern of individual cells
-    network_builder.cells[0].plot_voltage()
-    plt.title('%s (gid=%d)' % (cells[0].name, gid))
+dpls = simulate_dipole(net, n_trials=1)
+
+# The cell IDs (gids) are stored in the network object as a dictionary
+gid_dict = net.gid_dict
+print(net.gid_dict)
```

Comment:
not sure if you intended to create a new cell here with an output but if you want to do that, you can simply add:

###############################################################################